src/app/beer_garden/api/http/exceptions.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from tornado.web import HTTPError
+
+
+class BaseHTTPError(HTTPError):
```

Comment:
See #1146 for some info on this. I originally just kept following the same bad pattern here and then opened that issue to fix it everywhere, but decided to come back and just get things started.